package.json

```diff
@@ -50,6 +54,12 @@
           "default": false,
           "description": "If True, append initialContent to the existing file.",
           "order": 2
+        },
+        "tempfile.ending": {
```

Comment:
I think uses would want to use both feature:

* Create a file without any prompt
* Create a file with inputting extension

However, one global config value makes users either.

---

I think to prepare 2 commands is better without this new config value.

Reply:
My suggestion is if a user doesnt inputs anything in the prompt `.md` can be used as a default. The ux would then be double escape for default value. Possibly a additional cfg could be added here.

Reply:
In the updated code it will use the file ending of the file location from the allready existing config (so defaultly `.md`)